webauthn4j-metadata/src/main/java/com/webauthn4j/metadata/FidoMdsMetadataItemsProvider.java

```diff
@@ -222,7 +223,7 @@ MetadataStatement fetchMetadataStatement(String uri, byte[] expectedHash) {
         String metadataStatementBase64url = httpClient.fetch(uriWithToken);
         String metadataStatementStr = new String(Base64UrlUtil.decode(metadataStatementBase64url));
         byte[] hash = MessageDigestUtil.createSHA256().digest(metadataStatementBase64url.getBytes(StandardCharsets.UTF_8));
-        if (!Arrays.equals(hash, expectedHash)) {
+        if (!MessageDigest.isEqual(hash, expectedHash)) {
```

Comment:
`hash` is also known data to a metadata statement provider because it is provided by them too, there is no risk of timing attack here.